### PR TITLE
README.md: Update README.md to add link to tracker.ceph.com

### PR DESCRIPTION
**Pull request by johnbent**
I searched the existing documentation for a link to the tracker and had a hard time finding it. Other folks like myself might like it so prominently displayed.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -213,3 +213,6 @@ To build the documentation, ensure that you are in the top-level
 
 	admin/build-doc
 
+## Reporting Issues
+
+To report an issue and view existing issues, please visit https://tracker.ceph.com/issues.
```
Comment (ljflores) [inline README.md:218]: I would simply put "https://tracker.ceph.com", which would guide users to the home page. From there, they can navigate to the tab of their choice.

**Review by ljflores — CHANGES_REQUESTED** [review]
Thanks for your contribution @johnbent.

In addition to what I wrote below, please amend your commit to comply with the Ceph project's commit conventions:
1. Modify the title of your commit to be prefixed by the component you are modifying, in your case "README.md": `README.md: Update README.md to add link to tracker.ceph.com`. See https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#commit-title for more details.
2. Sign off with your name and email, like so: Signed-off-by: Random J Developer <random@developer.example.org>. See https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#developer-s-certificate-of-origin-1-1 for more details.

Let me know if you have any questions!

**Comment by johnbent** [comment]
Thanks @ljflores ! I don't know how to add a sign-off to a previous commit. Is it OK for me to just close this one and make the three requested changes in a new PR?

**Comment by ljflores** [comment]
> Thanks @ljflores ! I don't know how to add a sign-off to a previous commit. Is it OK for me to just close this one and make the three requested changes in a new PR?

@johnbent You can use `git commit --amend` to solve this. While on your branch (`patch-1`) use `git commit --amend` to edit your most recent commit. There, you can change the commit title and add a "Signed-off-by" line, and then force push the change to your branch: `git push -f <remote repository name> <branch name>`.

**Comment by johnbent** [comment]
Thank you very much @ljflores . I very much appreciate the kindness shown to a git newbie here! I believe I have successfully addressed your three requests:"
1. Used git commit --amend to add DCO
2. Used git commit --amend to change the title
3. Changed the link to https://tracker.ceph.com/issues

In terms of 3, however, I changed it to https://tracker.ceph.com/projects/ceph instead of to https://tracker.ceph.com as you suggested. I did that because I find that top-parent page to be a bit confusingly blank whereas /projects/ceph seems to be an appropriate landing place. Totally happy to change it to the top-parent however if you think that is best.

**Comment by ljflores** [comment]
> Thank you very much @ljflores . I very much appreciate the kindness shown to a git newbie here! I believe I have successfully addressed your three requests:"
> 
>     1. Used git commit --amend to add DCO
> 
>     2. Used git commit --amend to change the title
> 
>     3. Changed the link to https://tracker.ceph.com/issues
> 
> 
> In terms of 3, however, I changed it to https://tracker.ceph.com/projects/ceph instead of to https://tracker.ceph.com as you suggested. I did that because I find that top-parent page to be a bit confusingly blank whereas /projects/ceph seems to be an appropriate landing place. Totally happy to change it to the top-parent however if you think that is best.

@johnbent The changes to your commit look great! As for the link, I see your point about the home page. I think your idea of linking https://tracker.ceph.com/projects/ceph is a suitable solution.

There are some tests failing, but they aren't related to your change. I'm going to try and get them to pass before we merge this, but otherwise this is all good.

**Review by ljflores — APPROVED** [review]

**Comment by ljflores** [comment]
jenkins retest this please

**Comment by ljflores** [comment]
@neha-ojha do you think we should backport this to other stable versions? Or is the change minor enough to just merge into master?

**Comment by ljflores** [comment]
+1 from Neha, we can go ahead and merge this.

**Pull request merged**